src/content/managing-engineers.md

```diff
@@ -0,0 +1,22 @@
+- The way that engineers in teams work together is collaborative and open: we encourage people to voice their opinions and teams work together to find solutions to problems that everyone is happy with. The engineering manager in the team is ultimately accountable for the direction the team travels and makes sure that teams decisions are well considered and align with the goals of the organisation.
+
+- You and your line manager should schedule one to one meetings on a regular basis, typically fortnightly though the frequency will vary depending on how long you’ve been with the organisation or the team, and on if you have a  preference or a need for more frequent contact. These meetings are an opportunity for you to raise any concerns you may have, discuss your career progression, identify training and development needs and really anything else that is affecting your working life. It’s also an opportunity for your manager to give you feedback on your performance - things that are working well and also areas where you would benefit from a change.
+
+- Requests for holidays, notification of sickness and any changes for the way you work should go through your line manager, but it’s also a great idea to keep your team informed to ensure that your plans won’t cause a problem. As an organisation we like to say yes to reasonable requests but we also need to balance these against the needs of the business which your manager acts as a proxy for, and we need to ensure that everyone is treated fairly when dealing with these requests.
+
+- Your line manager is also responsible for your appraisals which are conducted on a quarterly basis - further details are available elsewhere. The same applies to career progression which you and your manager will collaborate on, this has a bi-annual cadence. In addition you may also write a personal development plan together which will take into account your longer term career aspirations and set some objectives which are designed to develop your skills so you can achieve these goals.
+
+**What’s the difference between an Engineering Manager, a Product Manager and a Scrum Master?**
+- **Engineering Managers** are accountable for the engineering performance of software systems, for the technical decisions made in the team and for line managing engineers - the “technical how” if you like.
+
+- **Product Managers** are responsible for the high level direction of the product and facilitate relationships between engineers and stakeholders. They generally own the “what” of the team.
+
+- **Scrum Masters** facilitate the smooth running of the team, helping to unblock issues, take a lead on external comms and organise team meetings - we could characterise this as the “process how”.
+
+**Still Confused? —** Just have a word with your line manager who should be able to talk you through all of the processes and procedures we have around management, or at the very least will find out and get back to you.
```

Comment:
This part reads like we're talking to one of our reports as opposed to someone reading this who is a prospective Guardian applicant/joiner.

Reply:
We could just drop this paragraph, or replace it with something more generic about how to find out more about working at the Guardian.

Reply:
I'm going to remove this section, I think this document is already quite long as it is.